src/griffe/agents/nodes.py

```diff
@@ -535,6 +535,7 @@ def parse__all__(node: NodeAssign) -> set[str]:  # noqa: WPS116,WPS120
         """
         return {elt.s for elt in node.value.elts}  # type: ignore[attr-defined]
 
+
```

Comment:
Useless change (and Black would not be happy), please remove :smile: 

```suggestion
```

Reply:
Actually black added this line !
I have version 21.9b

Reply:
I see, my bad!